### PR TITLE
add im-pipelines-utils

### PR DESCRIPTION
**Pull request by simonbray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by simonbray** [comment]
@tdudgeon, we are starting work on creating Galaxy tools for some of your pipeline scripts, as we discussed a few months back, and need to publish these packages on conda-forge.

Let me know if you or one of your colleagues would like to be added as a maintainer.

**Comment by simonbray** [comment]
Will make a separate PR for `im-pipelines-utils-rdkit` when this is merged.

**Comment by tdudgeon** [comment]
@simonbray Thanks for the notification. Alan Christie (@alanbchristie) would also be interested in this. Let us know if you want to discuss anything.

File: recipes/im-pipelines-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/InformaticsMatters/pipelines-utils/archive/pypi-{{ version }}.tar.gz
+  sha256: 70ddb4903a9f2f98d188b721b0f27850d9f7be660786ed64abc79a54c82ea03e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install src/python/ -vv"
```
Comment (marcelotrevisani) [inline recipes/im-pipelines-utils/meta.yaml]: Could it be ``noarch``?

```suggestion
  script: "{{ PYTHON }} -m pip install src/python/ -vv"
  noarch: python
```

**Pull request merged**